packages/super-components/components/msc-billing-tile/src/msc-billing-tile/menu-custom.scss

```diff
@@ -0,0 +1,39 @@
+.menu {
+  position: absolute;
+  display: flex;
+  flex-direction: column;
+  right: var(--ods-size-04);
+  align-items: flex-end;
+  top: 50%;
+
+  .triangle {
+    position: absolute;
+    right: 0;
+    top: var(--ods-size-n-05);
+    display: inline-block;
+    height: 0;
+    width: 0;
+    border-right: 9px solid transparent;
+    border-bottom: var(--ods-size-05) solid var(--ods-color-gray-000);
+    border-left: 9px solid transparent;
+    margin-right: 7px;
+  }
+
+  .tooltip {
+    position: relative;
+    filter: drop-shadow(0px 0px 5px var(--ods-color-gray-100));
+    border-radius: var(--ods-size-border-radius-02);
+    border: 1px solid var(--ods-color-gray-000);
```

Comment:
**nitpick:** Instead of hard-coding the CSS in pixels, maybe we can think of creating a utility classes for super-components and reuse them to maintain consistency across the super-components.

Reply:
This is temporary since ODS will provide a menu component